torchaudio/functional.py

```diff
@@ -683,6 +684,36 @@ def lowpass_biquad(waveform, sample_rate, cutoff_freq, Q=0.707):
     return biquad(waveform, b0, b1, b2, a0, a1, a2)
 
 
+def equalizer_biquad(waveform, sample_rate, center_freq, Q, gain):
+    # type: (Tensor, int, float, float, float) -> Tensor
+    r"""Designs biquad peaking equalizer filter and performs filtering.  Similar to SoX implementation.
+
+    Args:
+        waveform (torch.Tensor): audio waveform of dimension of `(channel, time)`
+        sample_rate (int): sampling rate of the waveform, e.g. 44100 (Hz)
+        center_freq (float): filter’s central frequency
+        Q (float): https://en.wikipedia.org/wiki/Q_factor
+        gain (float): desired gain at the boost (or cut)
```

Comment:
The units for gain are dB, right? 

What do you think of "attenuation" instead of "cut"?